scripts/deployTalentNFT.ts

```diff
@@ -0,0 +1,28 @@
+import { ethers, network } from "hardhat";
+
+import { deployTalentNFT } from "./shared";
+
+async function main() {
+  console.log(`Deploying TalentNFT one to ${network.name}`);
+  const [creator] = await ethers.getSigners();
+  console.log(`Owner will be ${creator.address}`);
+  const nftContract = await deployTalentNFT(creator.address, "TalentNFT");
+  console.log(`TalentNFT Address: ${nftContract.address}`);
+
+  // @TODO: change to a better URI
```

Comment:
is this TODO still relevant?

Reply:
No, I can clear it out, this is already the proper metadata :)